argusclient/model.py

```diff
@@ -419,9 +419,11 @@ class Notification(BaseEncodable):
     PAGER_DUTY = "com.salesforce.dva.argus.service.alert.notifier.PagerDutyNotifier"
     REFOCUS_BOOLEAN = "com.salesforce.dva.argus.service.alert.notifier.RefocusBooleanNotifier"
     REFOCUS_VALUE = "com.salesforce.dva.argus.service.alert.notifier.RefocusValueNotifier"
+    SLACK = "com.salesforce.dva.argus.service.alert.notifier.SlackNotifier"
```

Comment:
There is class level documentation that could be updated with the latest list. It looks like it went out of sync even before your change, but you may please update it with the full list.